release/release_gold.sh

```diff
@@ -305,4 +308,47 @@ function test_boms {
 	fi
 }
 
+function update_release_info_date {
+	if [[ "${_PRODUCT_VERSION}" != *q* ]] ||
+	   [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 3)" -eq 0 ]] ||
+	   [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 1)" -lt 2024 ]]
+	then
+		lc_log INFO "Skipping the release info update."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	if [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 1)" -eq 2024 ]] &&
+	   [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 2 | tr -d q)" -le 2 ]] &&
+	   [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 3)" -le 11 ]]
+	then
+		lc_log INFO "Skipping the release info update."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	if [[ ! " ${@} " =~ " --test " ]]
+	then
+		lc_cd "${_PROJECTS_DIR}/liferay-portal-ee"
+
+		local product_group_version="$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 1,2)"
+
+		local quarterly_release_branch_name="release-${product_group_version}"
+
+		git branch --delete "${quarterly_release_branch_name}" &> /dev/null
+
+		git fetch --no-tags upstream "${quarterly_release_branch_name}":"${quarterly_release_branch_name}" &> /dev/null
+
+		git checkout "${quarterly_release_branch_name}" &> /dev/null
+
+		sed -i -e "s/release.info.date=.*/release.info.date=$(date -d @"${LIFERAY_RELEASE_RC_BUILD_TIMESTAMP}" +"%B %-d, %Y")/" release.properties
```

Comment:
@kiwm, can you please try to use just the parameter `-i`? If not, try to sort it.